app/views/templates/package_action.erb

```diff
@@ -0,0 +1,46 @@
+<%#
+kind: job_template
+name: Package Action - SSH Default
+job_name: PackageAction
+provider_type: Ssh
+template_inputs:
+- name: pre_script
+  description: A script to run prior to the package action
+  input_type: user
+  required: false
+- name: action
+  description: 'The package action: install, update, or remove'
+  input_type: user
+  required: true
+  options: "install\nupdate\nremove"
+- name: package
+  description: The name of the package, if any
+  input_type: user
+  required: false
+- name: post_script
+  description: A script to run after the package action
+  input_type: user
+  required: false
+%>
+
+# Pre Script
+<%= input("pre_script") %>
+PRE_RETVAL=$?
+
+# Action
+<% if @host.operatingsystem.family == 'Redhat' %>
+yum -y <%= input("action") %> <%= input("package") %>
+<% elsif @host.operatingsystem.family == 'Debian' %>
+apt-get -y <%= input("action") %> <%= input("package") %>
+<% end %>
```

Comment:
I've hit an issue with this as well, when running a job against unmanged host (and even with managed host, one is able to get into state where the operating system is not set). 

The question might be if, in this case, we need to use the server parameters, or rather writing bash conditions for checking, which command is available? Opinions?